database/schema.sql

```diff
@@ -0,0 +1,45 @@
+CREATE DATABASE carousel;
+
+\connect carousel;
+
+  CREATE TABLE places (
+    placeId SERIAL PRIMARY KEY,
+    title VARCHAR(30) NOT NULL,
+    pictureUrl VARCHAR (500) NOT NULL,
+    zipCode INTEGER NOT NULL,
+    typeOfRoom VARCHAR(30) NOT NULL,
+    bedsNumber SMALLINT NOT NULL,
+    rating real NOT NULL,
+    totalReview INTEGER NOT NULL,
+    plusHost BOOLEAN,
+    superHost BOOLEAN,
+    price INTEGER NOT NULL,
+    link VARCHAR(500) NOT NULL
+  );
+
+  CREATE TABLE users (
+    userId SERIAL PRIMARY KEY,
+    userName VARCHAR(30) NOT NULL
+  );
+
+  CREATE TABLE user_lists (
+    listId SERIAL PRIMARY KEY,
+    listName VARCHAR(30) NOT NULL,
+    userId INTEGER NOT NULL REFERENCES users ON DELETE CASCADE
```

Comment:
I would add FOREIGN KEY here. If this works without FOREIGN KEY, then that is fine. I always used F KEY and I wasn't sure it works without one.